CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+# CHANGELOG
+
+All notable changes to this project will be documented in this file.
+
+## Unreleased
+
+## Alpha v0.5.0
+
+### Breaking Change
+
+* [#265](https://github.com/open-telemetry/opentelemetry-ruby/pull/265) Remove component attribute. ([@dmathieu](https://github.com/dmathieu))
```

Comment:
I don’t think this is a breaking change - it just removes a span attribute added by some instrumentation.

Reply:
fair enough, makes sense...i moved it over to enhacements but also happy to just not have it in changelog if not necessary. Alternatively maybe makes sense to add a `## Other` catchall subsection for stuff that doesn't fit neatly into the current subsections?

Reply:
I think this is OK in the enhancements section.